crates/aiken-project/src/blueprint/validator.rs

```diff
@@ -13,44 +14,36 @@ use serde;
 use uplc::ast::{DeBruijn, Program, Term};
 
 #[derive(Debug, PartialEq, Clone, serde::Serialize, serde::Deserialize)]
-pub struct Validator<R, S> {
+pub struct Validator {
```

Comment:
Parameters no longer needed now that we `serde::Deserialize` is implemented for `Schema`